scripts/Gossip.js

```diff
@@ -40,28 +40,42 @@ var Gossip = function(robot) {
     });
   }
 
-  var findGossip = function(text, msg) {
+  var findGossip = function(text, msg, limiter) {
     var connection = connectionHelper.getConnection();
     var query = 'SELECT * FROM gossip';
     var queryArgs = [];
+    var gossip = '\n';
 
     if (text != null && text.length > 0) {
       query += ' WHERE gossip_text LIKE ?';
       queryArgs.push('%' + text + '%');
     }
 
-    query += ' ORDER BY RAND() LIMIT 0,1';
+    query += ' ORDER BY RAND() LIMIT ' + (limiter || 1);
```

Comment:
This statement is weird.  You're doing an concat-assignment with a concat'd string, and the `(limiter || 1)` statement is possibly an integer.  Given Javascript does all manner of fuckery, I'd try to restructure this so it's less prone to problems.

Maybe like 
```
var order_by_clause = ' ORDER BY ... etc';
query = select_clause + from_clause + where_clause + order_by_clause;  
```
etc